packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -34,57 +32,33 @@ export const pageQuery = graphql`
 
 <p>{props.pageContext.frontmatter.description}</p>
 
-<Box marginTop="space100" marginBottom="space100">
-  <StatusLegend />
-</Box>
-
 <Box>
 
-## Release 2020.02.25
-
-Release date: February 25, 2020
+  ## Upcoming Release
 
-**Design releases**
+  Paste's upcoming release will be on **April 7, 2020**. This is what we've planned:
 
-- Design more form field components (Select, Checkbox and Checkbox group, and Radio and Radio Group). 📝
-- Design a Form composition component. 📝
-- Design a Toast component. 🍞
+  - Form components are the bread and butter for most Twilio experiences. With this release we'll ship seven components (Text Input, Textarea, Select, Checkbox, Checkbox Group, Radio, and Radio Group) for Sketch and React. 📝 
```

Comment:
"Form components are the bread and butter for most Twilio experiences. With this release we'll ship three form components (Text Input, Textarea, Select) for Sketch and React."